content/cookbook/20-rsc/20-stream-text.mdx

```diff
@@ -20,14 +20,14 @@ Text generation can sometimes take a long time to complete, especially when you'
 
 ## Client
 
-Let's create a simple React component that will call the `generate` function when a button is clicked. The `generate` function will call the `streamText` function, which will then generate text based on the input prompt. To consume the stream of text in the client, we will use the `readStreamableValue` function from the `ai/rsc` module.
+Let's create a simple React component that will call the `generate` function when a button is clicked. The `generate` function will call the `streamText` function, which will then generate text based on the input prompt. To consume the stream of text in the client, we will use the `readStreamableValue` function from the `@ai-sdk/react/rsc` module.
```

Comment:
wonder if we need to split it out into `@ai-sdk/react/rsc` or if we can integrate directly into `@ai-sdk/react`? (no need to change in this PR, can be a separate PR if we decide to do this)